tests/test_recipient_validation.py

```diff
@@ -138,6 +138,7 @@
     'spaces-in-domain@dom ain.com',
     'underscores-in-domain@dom_ain.com',
     'pipe-in-domain@example.com|gov.uk',
+    'comma,in-domain@gov.uk',
```

Comment:
This is `comma,in-local` technically (but functionally it doesn’t make a difference).

Reply:
Thanks - now updated.